### ISSUE TITLE
Mame 078 and imame4all individual themes

### ISSUE THREAD

**Issue by DarthNeo2** [digitalLumberjack/recalbox-os#483]
First of all, thank you for added MAME 0.78.

I wonder if it is possible to add a separate theme only to the imame4all and one for mame 078, like the two FBA. The advantage would be better to take advantage of the compatibility of each without having to change the Core in recalbox.conf. I use a rasp2 and MK 1 and 2 run very smoothly in imame4all but would not want to lose mame 078 or having to choose just one at a time.

Please it would be great for everyone.

**Comment by digitalLumberjack** [comment]
Feel free to create your themes and provide it to the community :)
https://github.com/digitalLumberjack/recalbox-os/wiki/Edit-emulationstation-themes-%28EN%29

**Comment by DarthNeo2** [comment]
Thanks for the answer, but that's not what I need. I could already apply a separate theme for mame2003 and one for the imame4all but can not make the games begin using the core, when the game starts is a black screen and then returns to the FrontEnd. I want the theme of the 2003 start with its Core and the theme imame4all start with your core, use the beta14 version and can only start games in mame 2003 is the default Core.

Please, if it is not possible to add it in future releases at least tell me how to do. Thanks in advance.

**Comment by rockaddicted** [comment]
I don't think than we'll do that.
Imame4all won't be used anymore, and you can switch on it if you really want via recalbox.conf

I tried Mortal Kombat 1 & 2 with the libretro core mame2003, and I play without any problems.
The games are fullspeed, no sound scratchs...
Did you update your romset from 0.37b5 (used with imame4all) to 0.78 (used with mame2003) ?

**Comment by DarthNeo2** [comment]
Yes, i downloaded the romset mame 0,78 only for this. The sound in mk1 and fps are very lower than before in imame4all Mame2003 have a far better compatibility but i need mk1 running 100%.
There is a way to configure the Core only for a specific game?

**Comment by rockaddicted** [comment]
But MK1 run very well on my core mame2003, as others games.
I don't need anymore imame4all and romset 0.37b5

If MK1 don't work with mame2003 on your recalbox, you have a problem with your MK1 rom.

**Comment by DarthNeo2** [comment]
@rockaddicted 

Thank you for quick response. 
You could send me your rom mk1 or tell me where you downloaded? My rom I got into a torrent site, i download the full mame 078 romset.

**Comment by rockaddicted** [comment]
Hum yes I'm using overclock (preset included in emulationstation's option menu).
And in libretro-mame2003 core options, there is an option "speedkack MK2/MK3".
I don't know if it is enabled by default. You can give a look.
To find core options, hotkey+b in game, then quick menu/core options/speedkack MK2/MK3

**Comment by DarthNeo2** [comment]
@rockaddicted 

I tried the speed hack but is not for mk1. I was using before the IMame4all without overclocking and functioned perfectly, I believe I will need to overclock, my only problem with this is that whenever I use this option, my PI2 crashes even with sinks. Is there a way to use a smaller overclock? Perhaps 950MHz or 1000MHz solve my problem.

Thank you for the patience.
